backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/snapshots/ColdMergeSnapshotSingleDiskCommand.java

```diff
@@ -113,6 +115,14 @@ public boolean performNextOperation(int completedChildCount) {
         }
     }
 
+    private RemoveSnapshotSingleDiskStep getInitialMergeStepForImage() {
+        if (getImageInfoFromVdsm(getDestinationDiskImage()) == null) {
```

Comment:
does GetVolumeInfo prepare the image on the vdsm side? if not, we may need to do that beforehand

Reply:
Tried to run cold snapshot remove, and it seems that getVolumeInfo works well even when the volume is not mounted